src/lib.rs

```diff
@@ -93,6 +93,9 @@ impl Cache {
         // Remove `"hub"`
         path.pop();
         path.push("token");
+        if !path.exists() {
+            log::warn!("File with your token is absent by path {:?}", path);
```

Comment:
Can we move that log to `api::{sync,tokio}` instead in the error branch while loading from this path ?

This function could be called many times over spamming every caller.

The token path is always the token path, it shouldn't ever warn.